cs2pr

```diff
@@ -58,8 +87,11 @@ foreach ($root as $file) {
         $line = (string) $error['line'];
         $message = (string) $error['message'];
 
-        annotateCheck(annotateType($type), relativePath($filename), $line, $message);
-        $exit = 1;
+        annotateCheck(annotateType($type), relativePath($filename), $line, $message, $colorize);
+
+        if (!$gracefulWarnings || $type === 'error') {
```

Comment:
at this point `$type` could also be `failure`.

the check here should be based on the result of `annotateType($type)` not `$type` itself.

Reply:
Is the `failure` an official value? However, I've changed the code as proposed and adopted the test. :-)

Reply:
Great thx. „Failure“ is a  value the lib cs2pr takes inspiration of supported.  Never did research whether this is a official value :-)

Thx